src/Maestro/Maestro.Web/Pages/Index.cshtml.cs

```diff
@@ -29,22 +30,24 @@ public PageResult OnGet()
 
         public HtmlString GetStyleBundles()
         {
-            var angularGeneratedIndex = Path.Join(Environment.ContentRootPath, "angular-bundle.html");
-            var fileText = System.IO.File.ReadAllText(angularGeneratedIndex);
-            var firstLink = fileText.IndexOf("<link", StringComparison.Ordinal);
-            var endOfHead = fileText.IndexOf("</head>", StringComparison.Ordinal);
-            var links = fileText.Substring(firstLink, endOfHead - firstLink);
-            return new HtmlString(links);
+            var assetsJson = Path.Join(Environment.WebRootPath, "assets.json");
+            var assets = JObject.Parse(System.IO.File.ReadAllText(assetsJson));
+
+            var links = assets["styles"].ToObject<JArray>()
```

Comment:
>.ToObject<JArray>() [](start = 40, length = 19)

Weird...  Is this necessary?

Reply:
Its a JToken thing, I think it could probably be a cast? This does the same thing though.